jenkins/job-configs/global.yaml

```diff
@@ -7,6 +7,10 @@
                     mkdir -p _tmp
                     curl -fsS --retry 3 "https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/upload-to-gcs.sh" > ./_tmp/upload-to-gcs.sh
```

Comment:
it's slightly unfortunate that if this fails, we get no build log at all now. not sure how to deal with that.